docs/pages/setup/reference/terraform-provider.mdx

```diff
@@ -313,12 +313,10 @@ Spec contains parameters of a resource.
 | `client_secret`                     | string      | GitHub OAuth app client secret                                  |
 | `display`                           | string      | Connector display name                                          |
 | `redirect_url`                      | string      | Authorization callback URL                                      |
-| `teams_to_logins`                   | set         | Maps GitHub team memberships onto allowed logins/roles.
-| `teams_to_logins.kubernetes_groups` | string list | List of allowed kubernetes groups for this org/team             |
-| `teams_to_logins.kubernetes_users`  | string list | List of allowed kubernetes users to impersonate for this org/team |
```

Comment:
What happened to kube users/groups? How are they configured with GitHub SSO now?

Reply:
You'd accomplish this by giving access via roles. I hope I made that sufficiently clear within the original PR (might have  been not obvious).